cloud/security.rst

```diff
@@ -126,3 +126,31 @@ All data within Citus Cloud is encrypted at rest, including data on the instance
   <script type="text/javascript">
   analytics.track('Doc', {page: 'Security', section: 'cloud'});
   </script>
+
+Network perimeter controls
+==========================
+
+All Citus clusters run in their own isolated Amazon Virtual Private Cloud (VPC). In addition to all security guarantees which are on by default you can also secure your citus cluster by limiting network access. There are two methods for limiting network access:
+
+1. VPC peering
+2. IP whitelisting
+
+VPC Peering
+-----------
+
+A VPC peering connection is a networking connection between two VPCs that
+enables you to route traffic between them using private IPv4 addresses or IPv6
+addresses. Instances in either VPC can communicate with each other as if they
+are within the same network. In order to setup a VPC peering connecting between
+your existing Citus cloud formation and your existing Amazon VPC open a support
```

Comment:
Probably change `Citus cloud` to `Citus Cloud` since we use it that way elsewhere I think